Project-2/Problem-2e/README.md

```diff
@@ -0,0 +1,18 @@
+# Problem2e implementation
+
+Here we for the first time probe the NQS wavefunction on a system with Coulomb repulsion.
+
+# Building the code
+If run from the root directory of the Git-repo, the build commands are:
+
+```
+make app=Project-2/Problem-2e/probe_nqs_repulsive
+```
+
+# Running simulations
+
+Sample invokation
+```
+./bin/Project-2/Problem-2e/importance_sampling_vs_brute_force.out 2 2 50 100
```

Comment:
Should be probe_nqs_repulsive.out here right?

Reply:
Fixed